frontend/src/components/AppHeader.tsx

```diff
@@ -7,13 +7,19 @@ type Props = {
   description?: string | ReactElement;
   contentRight?: React.ReactNode;
   breadcrumb?: boolean;
+  hideTrigger?: boolean;
```

Comment:
is this the best way? or can we not use the App header from the CreateNodeMigrationFileSuccess page?

PS. this variable naming is vague - what trigger?

Reply:
I thought of doing that but went with less code option. Wdyt? `showSidebarTrigger` or adding header?

Reply:
It is only visible in mobile, because sidebar is present by default in >md widths